src/App.js

```diff
@@ -7,6 +7,9 @@ import { AuthProvider } from 'contexts/auth';
 import RouteList from 'routes';
 
 function App() {
+  useEffect(() => {
```

Comment:
@rafayet-monon Do we need this block? 🤔

Reply:
Hell no, I was testing something :p Fixed in [149b5a1](https://github.com/rafayet-monon/surveyor/pull/18/commits/149b5a17fbca734b0f1aac5997d84f734bf1ceb7)